animation_nodes/data_structures/midi/midi_track.py

```diff
@@ -0,0 +1,49 @@
+from . midi_note import midiNote
+from ... data_structures import DoubleList
+
+class midiTrack:
+    def __init__(self, trackIndex, trackName):
+        self.index = trackIndex
+        self.name = trackName
+        self.notes = []
+
+    def addNote(self, note):
+        self.notes.append(note)
```

Comment:
The notes are added in order one by one when they are encountered in a track when opening the midi file. It is the MidiParseFile method which calls the addNote method.

Reply:
Do we want all the notes of a track to be added at once? That is to say constituted as a list in MidiParseFile and the list sent to the creation of the track in midi_track.py?

Reply:
In the parser method, just do `track.notes.append(note)`. I don't want this `addNote` method because it can be considered a public API where the user can add their own notes. Which I don't want because I want the notes to be ordered at all times.